keyboards/projectkb/alice/info.json

```
@@ -74,7 +74,7 @@
                 {"label":"Alt", "x":5, "y":4.25, "w":1.5},
                 {"label":"Space", "x":6.5, "y":4.25, "w":2},
                 {"label":"Menu", "x":8.5, "y":4.25},
-                {"label":"Space", "x":10.25, "y":4.25, "w":2.25},
+                {"label":"Space", "x":10.25, "y":4.25, "w":2.75},
                 {"label":"Alt", "x":12.5, "y":4.25, "w":1.5},
```

```suggestion
                {"label":"Alt", "x":13, "y":4.25, "w":1.5},
```

This key needs to be moved over as well, otherwise they overlap.

Handy tip: Press Ctrl+Shift+I in the Configurator and you can preview info.json files 😉

---

Noted Many Thanks.